src/components/ui/Checkbox.tsx

```diff
@@ -0,0 +1,43 @@
+'use client';
+
+/**
+ * This file is based on the Checkbox component from shadcn/ui and customized for our needs.
+ *  * The Checkbox component of shadcn/ui is based on  Checkbox component of Radix UI.
+ *
+ * See the official docs for more info:
+ * shadcn/ui: https://ui.shadcn.com/docs/components/checkbox
+ *  * Radix UI: https://www.radix-ui.com/themes/docs/components/checkbox
+ *
+ */
+
+import { CheckIcon } from '@/assets/images/icons';
+import { cn } from '@/lib/tailwind/utils';
+
+import { Indicator as CheckboxIndicator, Root as CheckboxRoot } from '@radix-ui/react-checkbox';
+import { ComponentPropsWithoutRef, ElementRef, forwardRef } from 'react';
+
+import { Icon } from '.';
+
+const Checkbox = forwardRef<
+  ElementRef<typeof CheckboxRoot>,
+  ComponentPropsWithoutRef<typeof CheckboxRoot>
+>(({ className, ...props }, ref) => (
+  <CheckboxRoot
+    ref={ref}
+    className={cn(
+      'peer',
+      'h-6 w-6 shrink-0 rounded-sm border border-primary',
```

Comment:
The current `border` width looks a bit too thin to me.
I tried `border-[1.2px]` and `border-[1.5px]`, and they both looked nicer.

In fact, the border width of the original svg image in Figma is around `1.2px`.

1px:
<img width="300" alt="image" src="https://github.com/genesis-tech-tribe/nishiki-frontend/assets/99148565/312b4bf6-832c-4030-ac62-09f07aa2593a">


1.2px (this is the closest value to the original design):
<img width="300" alt="image" src="https://github.com/genesis-tech-tribe/nishiki-frontend/assets/99148565/6f82ea35-0388-49aa-8102-bf687632b816">

1.5px:
<img width="300" alt="image" src="https://github.com/genesis-tech-tribe/nishiki-frontend/assets/99148565/fa931c65-df15-4eda-893b-a426e4dd4b28">

Reply:
Thank you for pointing out!! I was a bit blind about it, sorry 😓 

I didn't have much preference between `1.2px` and `1.5px` , so I decided to follow the figma, `1.2px`!
Thus I set it as `0.075rem` since we're using `rem` in our code base

Reply:
Using `rem` for such small values may not be effective... I have never seen that before.

I asked ChatGPT, and here is the conversation:
https://chat.openai.com/share/a85682c7-e98b-4ab9-9f84-a0a0edc9d88d

Could you research more about it, please?

Reply:
`Use px when you need fixed-size elements that won't change, such as for borders or shadows.`
Your suggestion and also this article below make sense. I think I'm going to use `px` for this
https://www.shecodes.io/athena/28078-when-to-use-ems-rems-and-px-in-css#:~:text=Use%20px%20when%20you%20need,their%20browser's%20default%20font%20size.

Other references:
https://stackoverflow.com/questions/33997943/0-1-rem-border-width-renders-differently-on-chrome-and-safari
https://kheohyeewei.medium.com/in-short-when-to-use-px-em-and-rem-b2de94e1b829

Reply:
Thank you for updating and providing additional information :)